planner/core/logical_plan_builder.go

```diff
@@ -749,8 +749,11 @@ func (b *PlanBuilder) coalesceCommonColumns(p *LogicalJoin, leftPlan, rightPlan
 	// Find out all the common columns and put them ahead.
 	commonLen := 0
 	for i, lName := range lNames {
+		if lName.ColName.L == "_tidb_rowid" {
```

Comment:
Check if column ID is `ExtraHandleID` instead?

Reply:
> Check if column ID is `ExtraHandleID` instead?

ID of column with name `_tidb_rowid`  is `-1`. I'm not sure whether there are other cases?